### PR TITLE
docs: Issue templates for bug and feature requests

### PR DESCRIPTION
**Pull request by jfuss**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Expected result
+
+Describe what you expected.
+
+### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)
+
+1. OS:
+2. `sam --version`:
+
```
Comment (TheSriram) [inline .github/ISSUE_TEMPLATE/Bug_report.md]:
```suggestion
I think the version of docker could also be a good idea.
```

Comment (TheSriram) [inline .github/ISSUE_TEMPLATE/Bug_report.md]:
```suggestion
3. Docker version
```

Comment (TheSriram) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Think asking for docker version is also a good idea. Thoughts?

Comment (jfuss) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Good idea. Do we only care about the version or deeper information?

We can suggest one of two commands here:
```
$ docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
```
or 

```
$docker --version
Docker version 18.06.1-ce, build e68fc7a
```

I am inclined for `docker --version` for now. I don't see a need of the other extra info.

Comment (TheSriram) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: yeah ```docker --version``` sounds good to me too.

Comment (jfuss) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: Can you update the suggestion? I want to see how this new github feature works :)

**Pull request merged**